### ISSUE TITLE
Not working on Mac

### ISSUE THREAD

**Issue by baharestani** [stevium/iceterm#1]
The plugin is listed on Rider on Mac but apparantly it is only supported for Windows.

**Comment by stevium** [comment]
Yes, ConEmu is a Windows-only Terminal. But, unfortunately, I didn't find any way of specifying the target OS for the IntelliJ plugins.
